api/kyverno/v1/policy_types.go

```diff
@@ -55,8 +55,11 @@ type Spec struct {
 	// the admission review request (enforce), or allow (audit) the admission review request
 	// and report an error in a policy report. Optional. The default value is "audit".
 	// +optional
+	// +kubebuilder:validation:Enum=audit;enforce
 	ValidationFailureAction string `json:"validationFailureAction,omitempty" yaml:"validationFailureAction,omitempty"`
 
+	ValidationFailureActionOverrides []ValidationFailureActionOverride `json:"validationFailureActionOverrides,omitempty" yaml:"validationFailureActionOverrides,omitempty"`
```

Comment:
There should be a godoc here so the CRD has a `description`. Additionally this should be `+optional` imo